content/docs/04.5-devx/05-app-profile.md

```diff
@@ -93,4 +96,73 @@ spec:
 
 </WarningBox>
 
+# App Profile Macros
+
+Pack macros are the variables defined in the App Profile and are only resolved at the cluster deployment time. The app profile variables can be:
+
+* directly given by the developers while modelling the application profiles.
+
+* resolved during the app instatiation.
+
+* funnelled/inherited from the lower tiers of the profile.
+
+The variables are of two types:
+
+* [The Input Parameters](/devx/app-profile#inputparameters)
+
+* [The Output Parameters](/devx/app-profile#outputparameters)
+<br />
+
+## Input parameters
+
+What user can specify and will be consumed by the current tier. It can even use output variables from below tier. This parameter resolves to a value of the variable defined in App Profile tier parameters.yaml file.
```

Comment:
```suggestion
User-provided values that will be consumed by the current tier. It can even use output variables from below tier. This parameter resolves to a value of the variable defined in App Profile tier parameters.yaml file.
```
@resmaa what does this mean `It can even use output variables from below tier`. It's clear what this means.
I also made the sentence more active vs passive by stating, "User-provided values vs what a user can specify". We always want to use direct voice when possible.

Reply:
yes